### ISSUE TITLE
analytics use hash instead of publication ID

### ISSUE THREAD

**Issue by brettins** [ekitabu-LLC/ekitabu-readium-desktop-working#38]

**Comment by brettins** [comment]
Waiting on @mocyeg to update the production server with the new migrations and code before I can close this out.

**Comment by mocyeg** [comment]
ah sorry for the late response. I realized after we spoke that the migrations run automatically on every deployment so it should already be in production.

**Comment by brettins** [comment]
Hey @mocyeg - the database shows no new migrations run in the migration_versions table, and the tables don't have the new hash item. Looks like the migrations didn't run for some reason.

**Comment by mocyeg** [comment]
Heyo @brettins - looks like the migrations weren't running because [that build never actually succeeded](https://github.com/ekitabu-LLC/ereader-analytics-api/runs/832045601?check_suite_focus=true). I googled a bit and I think there was some old config in there, so I updated it (as per [this guide](https://github.com/doctrine/DoctrineMigrationsBundle/blob/master/UPGRADE.md)).

it seems to be working now - let me know if you run into problems!

**Comment by kevinmainairungu** [comment]
@mutterback @mocyeg I sent out the script to extract the publication ID a while back. Were you able to extract it @mocyeg ?

**Comment by mocyeg** [comment]
@kevinmainairungu - I did get the [hash script `crc.js`](https://github.com/ekitabu-LLC/repository-api/blob/master/container/crc.js) working in the repository

**Comment by mutterback** [comment]
@kevinmainairungu is this ready to close?

**Comment by kevinmainairungu** [comment]
@mutterback form today's call and the data we have been getting, it's still in review. We are still not getting the publication hash for some of the events.

- It can just be an app_open event but no publication was read

- It can also be the firewall issue. 

@brettins any thought on this data?

[analytics events - eKitabu Desktop.csv.zip](https://github.com/ekitabu-LLC/ekitabu-readium-desktop-working/files/5689580/analytics.events.-.eKitabu.Desktop.csv.zip)

**Comment by brettins** [comment]
At this point I think this is a bug and needs a bug report/issue according to the guidelines John laid out.  A task should not be re-marked as a bug. 

There are too many variables - different server versions, different app install versions, etc., so without a proper bug report I won't be able to fix this.

**Comment by kevinmainairungu** [comment]
@brettins agreed. I'll be working with @dobbz on creating a proper bug report. This also touches on the database sync issue we have been having.
